### PR TITLE
Add pageRequests to private/public endpoints

### PR DESCRIPTION
**Pull request by dride-for-sure**

### CODE REVIEW

File: backend/src/main/java/org/opensource/schulaltas/model/school/RestPage.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ public RestPage (@JsonProperty ("content") List<T> content,
+                  @JsonProperty ("number") int number,
+                  @JsonProperty ("size") int size,
+                  @JsonProperty ("totalElements") Long totalElements,
+                  @JsonProperty ("pageable") JsonNode pageable,
+                  @JsonProperty ("last") boolean last,
+                  @JsonProperty ("totalPages") int totalPages,
+                  @JsonProperty ("sort") JsonNode sort,
+                  @JsonProperty ("first") boolean first,
+                  @JsonProperty ("numberOfElements") int numberOfElements) {
```
Comment (jamarob) [inline backend/src/main/java/org/opensource/schulaltas/model/school/RestPage.java:25]: Why these parameters, if they are not used in the constructor?

File: backend/src/test/java/org/opensource/schulaltas/controller/PrivateSchoolControllerTest.java [hunk 2]
```diff
@@ -103,18 +105,52 @@ private String getJWTToken () {
...
+
+  // THEN
+  assertThat( response.getStatusCode(), is( HttpStatus.OK ) );
+  assertThat( response.getBody().getContent(), is( List.of( getSchool( "2" ), getSchool( "1" ) ) ) );
+ }
+
+ @Test
+ @DisplayName ("List schools paginated sorted desc by number order should return a page of " +
+                       "schools sorted by number in desc order")
+ void listSchoolsPaginatedAsc () {
```
Comment (jamarob) [inline backend/src/test/java/org/opensource/schulaltas/controller/PrivateSchoolControllerTest.java:142]:
```suggestion
 void listSchoolsPaginatedDesc () {
```

**Review by jamarob — APPROVED** [review]
Well done 🎉 I have nothing to add besides a question and typo fix :)

File: backend/src/test/java/org/opensource/schulaltas/controller/PrivateSchoolControllerTest.java (hunk 2 above)
Comment (dride-for-sure) [reply backend/src/test/java/org/opensource/schulaltas/controller/PrivateSchoolControllerTest.java:142]: Wops! Merci.

File: backend/src/main/java/org/opensource/schulaltas/model/school/RestPage.java (hunk 1 above)
Comment (dride-for-sure) [reply backend/src/main/java/org/opensource/schulaltas/model/school/RestPage.java:25]: Here I have to test again. Without jackson could not map this correctly. All except `last`, `totalPages`,  `sort`,  `first` and `numberOfElements` are used in a constructor. Interesting is, that the RestPage includes these fields when used as type of a responseEntity ... I'll dig in here again!

**Comment by dride-for-sure** [comment]
> Well done 🎉 I have nothing to add besides a question and typo fix :)

Merci! See you tomorrow.

**Pull request merged**